plugins/spi/SPIOutputTest.cpp

```diff
@@ -83,30 +87,39 @@ void SPIOutputTest::setUp() {
 void SPIOutputTest::testDescription() {
   FakeSPIBackend backend(2);
   SPIOutput output1(m_uid, &backend, SPIOutput::Options(0, "Test SPI Device"));
+
   SPIOutput::Options options(1, "Test SPI Device");
   options.pixel_count = 32;
   SPIOutput output2(m_uid, &backend, options);
 
+  // check default constructor values
   OLA_ASSERT_EQ(
       string("Output 0, WS2801 Individual Control, 75 slots @ 1."
              " (707a:00000000)"),
       output1.Description());
   OLA_ASSERT_EQ(static_cast<uint16_t>(1), output1.GetStartAddress());
-  OLA_ASSERT_EQ(static_cast<uint8_t>(1), output1.GetPersonality());
```

Comment:
Perhaps have both checks here? At some point we should do a spot check that at least one of the personalities maps to it's old ID, and also that they're valid personality IDs.